4-Express/client/scripts/communication_manager.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios").default;
+const SERVER_URL = "http://localhost:3000";
+
+function addOrUpdateItem(id, item) {
+  const urlWithId = `${SERVER_URL}/${id}`;
+
+  axios.post(urlWithId, item).then(
```

Comment:
return the result of `axios.post` from this function so whoever consumes this api will wait for the promise to be fullfilled and not the function itself.
e.g:
`return axios.post(...).catch(withErrorHandling)`
and the part which consumes it
`addOrUpdateItem(...).then(/*do something*/)`